package.json

```
@@ -40,7 +40,7 @@
     "eslint-plugin-prettier": "3.3.1",
     "eslint-plugin-sort-imports-es6-autofix": "0.5.0",
     "eslint-plugin-sort-requires": "2.1.0",
-    "eslint-plugin-swg-internal": "file:build-system/eslint-rules",
+    "eslint-plugin-swg-internal": "github:subscriptions-project/eslint-plugin-swg#130d2940956eadbd568e8bebbffb43bedb9f7aa7",
```

```suggestion
    "eslint-plugin-swg-internal": "github:subscriptions-project/eslint-plugin-swg",
```

Could we point to the latest commit, to simplify things?

---

@ChrisAntaki Totally. In an earlier version I just pointed it to the #main branch, and can do so again.

---

That would rock :+1: